tests/dp_computations_test.py

```diff
@@ -657,5 +657,102 @@ def test_compute_dp_sum_noise_std_gaussian(self, eps: float, delta: float,
         self.assertAlmostEqual(scale, expected_std)
 
 
+class AdditiveMechanismTests(parameterized.TestCase):
+
+    @parameterized.parameters(
+        dict(epsilon=2, l1_sensitivity=4.5, expected_noise=2.25),
+        dict(epsilon=0.1, l1_sensitivity=0.55, expected_noise=5.5),
+    )
+    def test_laplace_mechanism_creation(self, epsilon, l1_sensitivity,
+                                        expected_noise):
+        mechanism = dp_computations.LaplaceMechanism(
+            epsilon=epsilon, l1_sensitivity=l1_sensitivity)
+        self.assertEqual(mechanism.noise_kind, pipeline_dp.NoiseKind.LAPLACE)
+        self.assertAlmostEqual(mechanism.noise_parameter,
+                               expected_noise,
+                               delta=1e-12)
+        self.assertAlmostEqual(mechanism.std,
+                               expected_noise * math.sqrt(2),
+                               delta=1e-12)
+        self.assertEqual(mechanism.sensitivity, l1_sensitivity)
+        self.assertIsInstance(mechanism.add_noise(1000), float)
+
+    @parameterized.parameters(
+        dict(epsilon=2, l1_sensitivity=4.5, value=0, expected_noise_scale=2.25),
+        dict(epsilon=0.1,
+             l1_sensitivity=0.55,
+             value=1000,
+             expected_noise_scale=5.5),
+        dict(epsilon=0.001,
+             l1_sensitivity=10,
+             value=-100,
+             expected_noise_scale=10000),
+    )
+    def test_laplace_mechanism_distribution(self, epsilon, l1_sensitivity,
+                                            value, expected_noise_scale):
+        # Use Kolmogorov-Smirnov test to verify the output noise distrition.
+        # https://en.wikipedia.org/wiki/Kolmogorov-Smirnov_test
+        mechanism = dp_computations.LaplaceMechanism(
+            epsilon=epsilon, l1_sensitivity=l1_sensitivity)
+        expected_cdf = stats.laplace(loc=value, scale=expected_noise_scale).cdf
+        noised_values = [mechanism.add_noise(value) for _ in range(30000)]
+
+        res = stats.ks_1samp(noised_values, expected_cdf)
+        self.assertGreater(res.pvalue, 1e-4)
```

Comment:
Do you know what flakiness we expect here approximately?

Reply:
It shouldn't be more than 1e-4 per test (that's why it tested against 1e-4). So in total something like 4*4e-4 (since there are 4 tests).